templates/base.html

```diff
@@ -7,28 +7,13 @@
     <meta name="description" content="">
     <title>{% block title %}Imaging experiments tracking{% endblock %}</title>
 
-
     <!-- Bootstrap core CSS -->
```

Comment:
This comment is misplaced now. I guess we don't need it at all since it's obvious that we import Bootstrap CSS

Reply:
So it's left from me copy-pasting code which is a bad thing to do. Feel free to remove comments that have no explanatory value and just clutter the code